bindings/matrix-sdk-ffi/src/room.rs

```diff
@@ -529,10 +541,22 @@ impl RoomMembersIterator {
         self.chunk_iterator.len()
     }
 
-    fn next_chunk(&self, chunk_size: u32) -> Option<Vec<Arc<RoomMember>>> {
-        self.chunk_iterator
-            .next(chunk_size)
-            .map(|members| members.into_iter().map(RoomMember::new).map(Arc::new).collect())
+    fn next_chunk(&self, chunk_size: u32) -> Option<Vec<ExportedRoomMember>> {
+        self.chunk_iterator.next(chunk_size).map(|members| {
+            members
+                .into_iter()
+                .map(|m| ExportedRoomMember {
+                    user_id: m.user_id().to_string(),
+                    display_name: m.display_name().map(|s| s.to_owned()),
+                    avatar_url: m.avatar_url().map(|a| a.to_string()),
+                    membership: m.membership().clone().into(),
+                    is_name_ambiguous: m.name_ambiguous(),
+                    power_level: m.power_level(),
+                    normalized_power_level: m.normalized_power_level(),
```

Comment:
Do we need to map `suggested_role_for_power_level` too?

Reply:
I only mapped the ones that the client was using, see https://github.com/element-hq/element-x-android/pull/2322/files#diff-a9d5b3af1dd9bbfe13b60da0a15cbc0b07a479146d463da3c09c94716aeb76f3R27-R35

Reply:
It's somewhat cheap to add another one, and that makes us less dependent on what the EX clients do, so let's add it.